stopes/ui/seamlisten/react_app/src/components/FileExplorer.tsx

```diff
@@ -121,14 +121,14 @@ function useFileNavigate() {
 }
 
 const Files = (): JSX.Element => {
-  const [displayHelper, setDisplayHelper] = useState(false);
+  const [displayHelper, setDisplayHelper] = useState(false); 
```

Comment:
Doesn't the linter remove this trailing spaces? Did you install and run [pre-commit](https://pre-commit.com/) ?

Reply:
I'll get right on it